script.js

```diff
@@ -1,4 +1,205 @@
+// constants for game modes
+const NAME_MODE = "name";
+const CHOICE_MODE = "choice";
+const SPS_MODE = "sps";
+const MJP_MODE = "mjp";
+
+// constants for output strings
+const SPS_HEADER = `SCISSORS! ✌ PAPER! ✋ STONE! ✊`;
+const MJP_HEADER = `MUK 묵! ✊ JJI 찌! ✌ PPA 빠! ✋`;
+const DRAW_MSG = `It's a draw! The computer is not satisfied. Play again?<br><hr>`;
+const WIN_MSG = `You won! Congrats! The computer is angry now. Play again?<br><hr>`;
+const LOSE_MSG = `You lost! Oh well :( The computer is gloating in victory. Play again?<br><hr>`;
```

Comment:
since timesUserWon and other variables are also global, you can consider embedding them into your win/lose messages